src/main/groovy/de/dkfz/eilslabs/batcheuphoria/execution/ExecutionService.groovy

```diff
@@ -26,6 +28,8 @@ interface ExecutionService {
 
     ExecutionResult execute(String command, boolean waitForIncompatibleClassChangeError, OutputStream outputStream)
 
+    RestResult execute(RestCommand restCommand)
```

Comment:
Please remove it. As said, override execute(Command) and handle it properly in the RestEM